blocks/header/header.js

```diff
@@ -136,6 +137,7 @@ async function decorateLangPicker(langPicker) {
   });
 
   const a = document.createElement('a');
+  a.href = '#';
```

Comment:
Same question for lang picker, on original site it isn't an anchor while we have it as anchor.